core/tests/protractor_utils/users.js

```diff
@@ -102,9 +112,14 @@ var _completeSignup = async function(username) {
   await action.click('Agree to terms checkbox', agreeToTermsCheckbox);
 
   var registerUser = element(by.css('.protractor-test-register-user'));
-  await action.click('Register user button', registerUser);
 
-  await waitFor.pageToFullyLoad();
+  await waitFor.clientSideRedirection(async() => {
+    // Clicking register user button to trigger redirection.
```

Comment:
Click the "register user" button to trigger redirection.

(Please update the rest of your comments similarly.)

Reply:
done